lib/utils/matchMutate.js

```diff
@@ -0,0 +1,21 @@
+import { useSWRConfig } from 'swr';
```

Comment:
This is just copypasta from https://swr.vercel.app/docs/advanced/cache#mutate-multiple-keys-from-regex